metadata/targets.json

```diff
@@ -127,8 +131,8 @@
     "length": 7256
    }
   },
-  "version": 5,
-  "x-tuf-on-ci-expiry-period": 3650,
-  "x-tuf-on-ci-signing-period": 365
+  "version": 6,
+  "x-tuf-on-ci-expiry-period": 42,
+  "x-tuf-on-ci-signing-period": 28
```

Comment:
`targets` expires in 42 days, signing event starts in 14 days -- we can increase these once we see the first one or two signing events working